plugins/http.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: http
+spec:
+  version: "v0.1.0"  # Replace with your current version
+  homepage: https://github.com/sttts/kubectl-http
+  shortDescription: "Make HTTP requests with kubeconfig credentials"
```

Comment:
Adding caveats: field saying it requires httpie could be useful if the program does not check for it.